Source/Plugins/Core/com.equella.core/js/tsrc/search/components/OwnerSelector.tsx

```diff
@@ -0,0 +1,168 @@
+/*
+ * Licensed to The Apereo Foundation under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * The Apereo Foundation licenses this file to you under the Apache License,
+ * Version 2.0, (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as OEQ from "@openequella/rest-api-client";
+import * as React from "react";
+import { useState } from "react";
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  Grid,
+  IconButton,
+  List,
+  ListItem,
+  ListItemIcon,
+  ListItemSecondaryAction,
+  ListItemText,
+} from "@material-ui/core";
+import { AccountCircle } from "@material-ui/icons";
+import DeleteIcon from "@material-ui/icons/Delete";
+import UserSearch from "../../components/UserSearch";
+import { languageStrings } from "../../util/langstrings";
+
+interface OwnerSelectorProps {
+  /** The currently selected user or. */
```

Comment:
I think this is what you meant? Or have I misunderstood?
```suggestion
  /** The currently selected user or undefined. */
```

Reply:
Ah yes, mostly. Fixed up. :)